.github/ct.yaml

```diff
@@ -15,3 +15,4 @@ check-version-increment: true
 debug: true
 chart-repos:
   - stable=https://kubernetes-charts.storage.googleapis.com/
+  - bitnami=https://charts.bitnami.com
```

Comment:
For what reason do we need this additional repository?

Reply:
This will be needed by the Hawkbit chart. Actually this change can be removed from this PR.

Reply:
Is there a way we can confine this then to the hawkbit chart?